components/globals/card/Card.tsx

```diff
@@ -1,14 +1,44 @@
 import React from "react";
 
+export enum HeadingLevel {
+  H1 = "h1",
+  H2 = "h2",
+  H3 = "h3",
+  H4 = "h4",
+  H5 = "h5",
+  H6 = "h6",
+}
+
 interface CardProps {
   children?: React.ReactNode;
   icon?: JSX.Element;
   title?: string;
   content?: string;
+  heading?: HeadingLevel;
+  headingStyle?: string;
 }
 
 export const Card = (props: CardProps) => {
-  const { children, icon, title, content } = props;
+  const { children, icon, title, content, heading, headingStyle } = props;
+
+  function createTitle(
+    heading: HeadingLevel | undefined,
+    content: string | undefined
+  ): React.JSX.Element {
+    if (heading && content) {
+      return React.createElement(
```

Comment:
`createElement` is part of the [React legacy API](https://react.dev/reference/react/createElement) but as far as I know these API functions aren't going anyways